sirepo/srunit.py

```diff
@@ -940,12 +940,10 @@ def _combine_req(encoded_uri):
 
         def _must_be_http(uri):
             # POSIT: /auth- match like sirepo.js msgRouter and https?:
-            # for browser click on email msg. If there are headers,
+            # for browser click on email msg. If there is Authorization header,
```

Comment:
This approach does not work, because websockets don't get http headers so reverted this when I did other changes.